core/src/main/java/cz/xtf/core/config/XTFConfig.java

```diff
@@ -29,11 +29,21 @@
 
 	private static final Properties properties = new Properties();
 
+	private static final String XTF_TEST_PROPERTIES_CONFIG_PROPERTY_NAME = "config.xtf.test.properties.path";
+	private static final String XTF_GLOBAL_TEST_PROPERTIES_CONFIG_PROPERTY_NAME = "config.xtf.global.test.properties.path";
+
 	// Pre-loading
 	static {
-		testPropertiesPath = XTFConfig.getProjectRoot().resolve("test.properties");
-		globalPropertiesPath = XTFConfig.getProjectRoot().resolve("global-test.properties");
-
+		if (!System.getProperty(XTF_GLOBAL_TEST_PROPERTIES_CONFIG_PROPERTY_NAME,"").isEmpty()) {
```

Comment:
I'd say it will be way clearer to use:
`resolvePropertiesPath(System.getProperty(GLOBAL_TEST_PROPERTIES_PATH, "global-test.properties"))`

It's one liner with same functionality and for both (set path and default one) will be used same mechanics for resolution.